modules/environment/main.tf

```diff
@@ -0,0 +1,36 @@
+module "workspace" {
+  source = "../workspaces"
+
+  auto_apply            = true
+  file_triggers_enabled = true
+  name                  = var.config_name
+  organization          = var.organization
+  trigger_prefixes      = ["config/${var.config_name}.yaml"]
```

Comment:
`config/` probably should be parameterized

Reply:
This will be fixed in the next commit 👍